### PR TITLE
Updated the links. Kubelet flag's docs will render links now

### LINKED ISSUE #31584: Kubelet flag's docs should render links.
There are more than 100 links like
https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ , is not rendering in https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/

**Expectation**: Links in kubelet flag's docs should not be treated as text.

**Page to Update**: https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/

### PR DESCRIPTION
**Pull request by draksha22**
Fixes #31584

### CODE REVIEW

File: content/en/case-studies/peardeck/index.html [hunk 1]
```diff
@@ -1,87 +0,0 @@
----
```
Comment (mk46) [inline content/en/case-studies/peardeck/index.html:1]: Could you please specify the reason for removing this file?

File: content/en/case-studies/wink/index.html [hunk 2]
```diff
@@ -58,7 +58,9 @@ <h2>Impact</h2>
...
 
-<p>In 2015, Docker wasn't yet widely used, but as Klein points out, "it was certainly understood by the people who were on the frontier of technology. We started looking at potential technologies that existed. One of the limiting factors was that we needed to deploy multi-port non-http/https services. It wasn't really appropriate for some of the early cluster technology. We liked the project a lot and we ended up using it on other stuff for a while, but initially it was too targeted toward http workloads."</p>
+ 
+<p>At the time – in 2015 – Docker wasn't yet widely used, but as Klein points out, "it was certainly understood by the people who were on the frontier of technology. We started looking at potential technologies that existed. One of the limiting factors was that we needed to deploy multi-port non-http/https services. It wasn't really appropriate for some of the early cluster technology. We liked the project a lot and we ended up using it on other stuff for a while, but initially it was too targeted toward http workloads."</p>
+
```
Comment (mk46) [inline content/en/case-studies/wink/index.html:63]: Please undo the changes.

File: content/en/docs/reference/command-line-tools-reference/kubelet.md [hunk 3]
```diff
@@ -51,14 +51,14 @@ kubelet [flags]
 <td colspan="2">--address string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: 0.0.0.0 </td>
 </tr>
 <tr>
-<td></td><td style="line-height: 130%; word-wrap: break-word;">The IP address for the Kubelet to serve on (set to <code>0.0.0.0</code> or <code>::</code> for listening in all interfaces and IP families)  (DEPRECATED: This parameter should be set via the config file specified by the Kubelet's <code>--config</code> flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.)</td>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">The IP address for the Kubelet to serve on (set to <code>0.0.0.0</code> or <code>::</code> for listening in all interfaces and IP families)  (DEPRECATED: This parameter should be set via the config file specified by the Kubelet's <code>--config</code> flag. See <a href="https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/">here</a> for more information.)</td>
```
Comment (mk46) [inline content/en/docs/reference/command-line-tools-reference/kubelet.md]: It would be great if you replace `here` with `kubelet-config-file`.

**Comment by mk46** [comment]
Fixes #31584

File: content/en/case-studies/peardeck/index.html (hunk 1 above)
Comment (draksha22) [reply content/en/case-studies/peardeck/index.html:1]: someone in the community told me that this file is a clone of other issue and that issue is resolved that's why i removed it. Should i undo this file also?

File: content/en/case-studies/wink/index.html (hunk 2 above)
Comment (draksha22) [reply content/en/case-studies/wink/index.html:62]: Done.

File: content/en/docs/reference/command-line-tools-reference/kubelet.md [hunk 4]
```diff
@@ -51,14 +51,14 @@ kubelet [flags]
 <td colspan="2">--address string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: 0.0.0.0 </td>
 </tr>
 <tr>
-<td></td><td style="line-height: 130%; word-wrap: break-word;">The IP address for the Kubelet to serve on (set to <code>0.0.0.0</code> or <code>::</code> for listening in all interfaces and IP families)  (DEPRECATED: This parameter should be set via the config file specified by the Kubelet's <code>--config</code> flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.)</td>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">The IP address for the Kubelet to serve on (set to <code>0.0.0.0</code> or <code>::</code> for listening in all interfaces and IP families)  (DEPRECATED: This parameter should be set via the config file specified by the Kubelet's <code>--config</code> flag. See <a href="https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> for more information.)</td>
```
Comment (draksha22) [inline content/en/docs/reference/command-line-tools-reference/kubelet.md]: @mk46 should i have to change all the here to kubelet-config-file??

**Review by draksha22 — COMMENTED** [review]
i have made the possible chnages. if you want any other changes @mk46 kindly tell me.

File: content/en/docs/reference/command-line-tools-reference/kubelet.md (hunk 4 above)
Comment (mk46) [reply content/en/docs/reference/command-line-tools-reference/kubelet.md]: Yes.

File: content/en/case-studies/peardeck/index.html (hunk 1 above)
Comment (draksha22) [reply content/en/case-studies/peardeck/index.html:1]: sorry it's clear to me. should i again add this file or i just leave it as it is? Could you please specify.

**Comment by mk46** [comment]
@draksha22 Apart from https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/  there are some other links that are not rendering. Search `http:/` and `https:/`  in https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/ and update them too.

File: content/en/case-studies/peardeck/index.html (hunk 1 above)
Comment (mk46) [reply content/en/case-studies/peardeck/index.html:1]: Yes, Please add this file. If any further confusion ping me on slack( mk46 )

File: content/en/docs/reference/command-line-tools-reference/kubelet.md [hunk 5]
```diff
@@ -44,70 +44,72 @@ kubelet [flags]
...
 </tr>
 <tr>
-<td></td><td style="line-height: 130%; word-wrap: break-word;">Enables anonymous requests to the Kubelet server. Requests that are not rejected by another authentication method are treated as anonymous requests. Anonymous requests have a username of <code>system:anonymous</code>, and a group name of <code>system:unauthenticated</code>. (DEPRECATED: This parameter should be set via the config file specified by the Kubelet's <code>--config</code> flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.)</td>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">Enables anonymous requests to the Kubelet server. Requests that are not rejected by another authentication method are treated as anonymous requests. Anonymous requests have a username of <code>system:anonymous</code>, and a group name of <code>system:unauthenticated</code>. (DEPRECATED: This parameter should be set via the config file specified by the Kubelet's <code>--config</code> flag. See <a href="https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> for more information.)</td>
 </tr>
```
Comment (draksha22) [inline content/en/docs/reference/command-line-tools-reference/kubelet.md]: i have done the changes

**Review by draksha22 — COMMENTED** [review]
@mk46 i have made the specified changes. kindly review them.

File: content/en/case-studies/peardeck/index.html (hunk 1 above)
Comment (draksha22) [reply content/en/case-studies/peardeck/index.html:1]: hey i am unable to add the file again and it seems someone has made the changes to that file. what should i do now?

Comment (mk46) [reply content/en/case-studies/peardeck/index.html:1]: I think you missed https://kubernetes.io/docs/contribute/new-content/open-a-pr/#create-a-local-clone-and-set-the-upstream. PTAL.

**Comment by tengqm** [comment]
please keep the PR focused on one thing.
/hold

**Comment by zacharysarah** [comment]
@draksha22 👋🏻 Please feel free to `/reopen` when you're ready to continue.

/close
